nautobot/docs/development/core/index.md

```diff
@@ -0,0 +1,322 @@
+* [GitHub discussions](https://github.com/nautobot/nautobot/discussions) - The preferred forum for general discussion and support issues. Ideal for shaping a feature request prior to submitting an issue.
+
+GitHub's discussions are the best place to get help or propose rough ideas for
+new functionality. Their integration with GitHub allows for easily cross-
+referencing and converting posts to issues as needed. There are several
+categories for discussions:
+
+* **General** - General community discussion.
+* **Ideas** - Ideas for new functionality that isn't yet ready for a formal
+  feature request. These ideas are what will be in scope to review when moving work from _Future_ to _Near Term_ as stated in the previous section.
+* **Q&A** - Request help with installing or using Nautobot.
+
+## Contributing
+
+We welcome many forms of contributions to Nautobot. While we understand most contributions will commonly come from developers, we encourage others to contribute in the form of docs, tutorials, and user guides. If you have other ideas for contributing, don't hesitate to open an issue or have a discussion in one of the forums above.
+
+Please also take a chance to look at our [Wiki on GitHub](https://github.com/nautobot/nautobot/wiki/Work-Intake-&-Issue-Management) to review any specifics as to how we define and scope work as a community.
+
+### Reporting Bugs
+
+* First, ensure that you're running the [latest stable version](https://github.com/nautobot/nautobot/releases)
+  of Nautobot. If you're running an older version, it's possible that the bug has
+  already been fixed.
+
+* Next, check the GitHub [issues list](https://github.com/nautobot/nautobot/issues)
+  to see if the bug you've found has already been reported. If you think you may
+  be experiencing a reported issue that hasn't already been resolved, please
+  click "add a reaction" in the top right corner of the issue and add a thumbs
+  up (+1). You might also want to add a comment describing how it's affecting your installation. This will allow us to prioritize bugs based on how many users are affected.
+
+* When submitting an issue, please be as descriptive as possible. Be sure to
+  provide all information request in the issue template, including:
+    * The environment in which Nautobot is running
+    * The exact steps that can be taken to reproduce the issue
+    * Expected and observed behavior
+    * Any error messages generated
+    * Screenshots (if applicable)
+
+* Please avoid prepending any sort of tag (e.g. "[Bug]") to the issue title.
+  The issue will be reviewed by a maintainer after submission and the appropriate
+  labels will be applied for categorization.
+
+* Keep in mind that bugs are prioritized based on their severity and how much
+  work is required to resolve them. It may take some time for someone to address
+  your issue.
```

Comment:
Remove gratuitous hard-line-wraps:

```suggestion
* First, ensure that you're running the [latest stable version](https://github.com/nautobot/nautobot/releases) of Nautobot. If you're running an older version, it's possible that the bug has already been fixed.

* Next, check the GitHub [issues list](https://github.com/nautobot/nautobot/issues) to see if the bug you've found has already been reported. If you think you may be experiencing a reported issue that hasn't already been resolved, please click "add a reaction" in the top right corner of the issue and add a thumbs up (+1). You might also want to add a comment describing how it's affecting your installation. This will allow us to prioritize bugs based on how many users are affected.

* When submitting an issue, please be as descriptive as possible. Be sure to provide all information request in the issue template, including:
    * The environment in which Nautobot is running
    * The exact steps that can be taken to reproduce the issue
    * Expected and observed behavior
    * Any error messages generated
    * Screenshots (if applicable)

* Please avoid prepending any sort of tag (e.g. "[Bug]") to the issue title. The issue will be reviewed by a maintainer after submission and the appropriate labels will be applied for categorization.

* Keep in mind that bugs are prioritized based on their severity and how much work is required to resolve them. It may take some time for someone to address your issue.
```